django/cantusdb_project/static/js/melody_search.js

```diff
@@ -39,12 +38,12 @@ function melodySearch() {
     const resultsDiv = document.getElementById("resultsDiv");
 
     // get the drawArea position (absolute position, relative to the document)
-    const drawAreaRect = drawArea.getBoundingClientRect();
-    const docRect = document.body.getBoundingClientRect();
-    const drawAreaLeft = drawAreaRect.left - docRect.left;
-    const drawAreaRight = drawAreaRect.right - docRect.left;
-    const drawAreaTop = drawAreaRect.top - docRect.top;
-    const drawAreaBottom = drawAreaRect.bottom - docRect.top;
+    let drawAreaRect = drawArea.getBoundingClientRect();
+    let docRect = document.body.getBoundingClientRect();
+    let drawAreaLeft = drawAreaRect.left - docRect.left;
+    let drawAreaRight = drawAreaRect.right - docRect.left;
+    let drawAreaTop = drawAreaRect.top - docRect.top;
+    let drawAreaBottom = drawAreaRect.bottom - docRect.top;
```

Comment:
> Change drawArea coordinates on resize wo deleting current search

I'm curious about the "wo deleting current search" part of this commit message. Is there a gotcha here that could lead to the current query or search results being lost if someone makes a minor tweak? If so, could a comment be added in the code explaining the tricky thing?

Reply:
I will clarify...